web/src/tippyjs.js

```diff
@@ -244,6 +244,7 @@ export function initialize() {
             "#compose_top_right [data-tippy-content]",
             "#compose_top_right [data-tooltip-template-id]",
         ],
+        delay: LONG_HOVER_DELAY,
```

Comment:
Yes, it is intentional because we have a long hover delay for all the tooltips present in the messagebox. This tooltip was not having any delay, making it inconsistent with the other tooltips.

| Before | After |
|------------|---------|
| ![before](https://user-images.githubusercontent.com/66828942/229445193-9f403672-f449-47bc-b036-6deace388949.gif) | ![after](https://user-images.githubusercontent.com/66828942/229445233-01574f56-2a5e-42fa-bc88-122691df5771.gif) |

I have updated the PR description and commit message for the same, could you please review it again.

Reply:
OK, that makes sense to me!